Makefile

```diff
@@ -0,0 +1,11 @@
+cloud_e2e_test:
+	bats ./tests/cloud-consumes-data-from-nats.bats
```

Comment:
This test/file does not exist. Same comment as for previous non-existing configs/tests.

Reply:
I took cloud_e2e_test out